temporian/core/data/duration.py

```diff
@@ -27,6 +27,10 @@
 Duration = float
 
 
+def subseconds(value: float) -> Duration:

```

Comment:
`milliseconds`?

Reply:
I see subseconds was chosen because of Mathieu's comment below, but milliseconds seems like the correct word for it. Be careful to change it where it appears below too 👀